tests/Doctrine/ODM/MongoDB/Tests/Functional/TargetDocumentTest.php

```diff
@@ -18,6 +19,15 @@ public function testMappedSuperClassAsTargetDocument(): void
         $this->dm->persist($test->reference);
         $this->dm->flush();
     }
+
+    public function testTargetDocumentIsResolvable(): void
+    {
+        self::expectExceptionMessage("Target document class 'Doctrine\ODM\MongoDB\Tests\Functional\SomeInvalidClass' used in field 'reference' of class 'Doctrine\ODM\MongoDB\Tests\Functional\InvalidTargetDocumentTestDocument' does not exist.");
```

Comment:
Pro tip: use `expectException` with exception factories to not be impacted by exception message changes:
```suggestion
        self::expectException(MappingException::invalidTargetDocument(SomeInvalidClass::class, InvalidTargetDocumentTestDocument::class, 'reference'));
```

Note that the code suggestion will most likely need a couple of `use` statements.

Reply:
Thanks, good suggestion!